packages/app/src/systems/Vault/services/VaultServer.ts

```diff
@@ -160,11 +177,35 @@ export class VaultServer extends EventEmitter {
     vaultId,
     password,
   }: VaultInputs['exportVault']): Promise<string> {
-    await this.manager.unlock(password);
+    if (password) await this.manager.unlock(password);
```

Comment:
This is a security rule to avoid the export been call without a password even if the vault is unlocked. This if needs to be removed.